### ISSUE TITLE
Creating new project fails

### ISSUE THREAD

**Issue by IngarMa** [Puzzlepart/prosjektportalen#422]
[ ] Enhancement

[X ] Bug

[X ] Question

#### Environment

[ ] SharePoint Online

[ X] SharePoint 2013

[ ] SharePoint 2016

#### Summary

When creating a new project, after/when status is "legger på mal", the creation stops.

This is the error in ULS-viewer : 
Exception occured in scope Microsoft.SharePoint.WebPartPages.SPLimitedWebPartManager.ImportWebPart. Exception=Microsoft.SharePoint.WebPartPages.WebPartPageUserException: Filen du importerte, er ugyldig. Kontroller
 at filen er en beskrivelsesfil for webdeler (*.webpart eller *.dwp), og
 at den inneholder godkjent XML.    
 at Microsoft.SharePoint.WebPartPages.WebPartImporter.CreateXdoc(XmlReader reader)    
 at Microsoft.SharePoint.WebPartPages.WebPartImporter..ctor(SPWebPartManager manager, XmlReader reader, Uri webPartPageUri, SPWeb spWeb)    
 at Microsoft.SharePoint.WebPartPages.WebPartImporter.Import(SPWebPartManager manager, XmlReader reader, Boolean clearConnections, Uri webPartPageUri, SPWeb spWeb)    
 at Microsoft.SharePoint.WebPartPages.WebPartImporter.Import(SPWebPartManager manager, XmlReader reader, Boolean clearConnections, SPWeb spWeb)    
 at Microsoft.SharePoint.WebPartPages.SPWebPartManager.ImportWebPart(XmlReader reader, String& errorMessage)    
 at Microsoft.SharePoint.WebPartPages.SPLimitedWebPartManager.ImportWebPart_Client(String webPartXml)    
 at Microsoft.SharePoint.Client.ServerStub.InvokeMethodWithMonitoredScope(Object target, String methodName, XmlNodeList args, ProxyContext proxyContext, Boolean& isVoid)

#### Version

Please specify what version you are using: [ prosjektportalen-2.1.13 ]

#### Expected / Desired Behavior / Question
I would like a project-room:-)

#### Observed Behavior
Creating a new site stops before completed.

#### Steps to Reproduce
Sp2013, with fresh install of projectportal
The installation of the

**Comment by tarjeieo** [comment]
Hi,

That's unfortunate. Can you help us with
1. Your SharePoint server patch level?
2. Is your user profile service configured? We've seen a similar issue with the sitefeed web part when UPS is not configured.

-T

**Comment by IngarMa** [comment]
Hi,

1. the patch level = september 2017. 
2. UPS is configured again, still same issues. 

Seems it stops when status is: legger til filer og webdeler.

**Comment by tarjeieo** [comment]
Any ULS entries that gives more hints?

**Comment by olemp** [comment]
Could you attach a screenshot of the frontpage of the project where it fails @IngarMa?

**Comment by IngarMa** [comment]
![image](https://user-images.githubusercontent.com/32195694/31772059-97bf3c72-b4de-11e7-95e8-a811cf11b59d.png)
Heres the portal

And here is a screenshot for ulsviewer
![image](https://user-images.githubusercontent.com/32195694/31772091-b83fce58-b4de-11e7-99cb-f7cf09bf045c.png)

**Comment by olemp** [comment]
@IngarMa That is the portfolio level. Do you have a screenshot of the project site? You can find it under "Siste prosjekter". J

**Comment by IngarMa** [comment]
@olemp  sorry ;-)

![image](https://user-images.githubusercontent.com/32195694/31772418-e2bb6e66-b4df-11e7-8890-010d87198e3a.png)

**Comment by olemp** [comment]
@IngarMa: from the attached screenshot it seems that UPS is not set up correctly. The next webpart to be added to that page is the SiteFeed webpart.

**Comment by PrinsMild** [comment]
Did you unzip the file coprrectly before installing? I had a simular issue a while ago, finding the zip-file had to be unblocked.

**Comment by IngarMa** [comment]
@olemp Thanks, was some permissions on the ups missing.
This again prevented the script from activating the sitefeed on the top/home.

Now i get a 400 bad request after this 
![image](https://user-images.githubusercontent.com/32195694/31944886-d6f08c82-b8cd-11e7-804d-6bd29fb208a1.png)

I'll have to do some more digging 👍

**Comment by olemp** [comment]
Do you have some more details around the 400 bad request error?

**Comment by IngarMa** [comment]
@olemp so far this is it:

![image](https://user-images.githubusercontent.com/32195694/31945063-60d18276-b8ce-11e7-8aee-3f0a4de0dd8d.png)

**Comment by IngarMa** [comment]
Request: (POST:http://prosjektportal5/_api/site/rootweb/lists/getByTitle('Logg')/items

**Comment by olemp** [comment]
The `Logg` list should reside in the root web.

**Comment by IngarMa** [comment]
the site: http://prosjektportal5 was redeployed approx 1 hour ago. No logg list there. Should there be?

Also no logg list in the room it-selv.

The room:
![image](https://user-images.githubusercontent.com/32195694/31945713-3e22bdd8-b8d0-11e7-9f7a-0e2a61c82fca.png)

the top:

![image](https://user-images.githubusercontent.com/32195694/31945727-4cc0a530-b8d0-11e7-97ab-b306c9a4eec5.png)

notice anything missing? 
Couldn't find anything standing out, comparring to my own o365 tenant, going over and checking the lists.

**Comment by olemp** [comment]
It's a hidden list. Try navigating to it: http://prosjektportal5/lists/log

**Comment by IngarMa** [comment]
List is there. Empty though

![image](https://user-images.githubusercontent.com/32195694/31946514-4e35e310-b8d2-11e7-86be-ec5e553678db.png)

**Comment by olemp** [comment]
Yeah, that's because POST to `http://prosjektportal5/_api/site/rootweb/lists/getByTitle('Logg')/items` is failing in your environment.

**Comment by IngarMa** [comment]
Ser denne i en correlation: 
![image](https://user-images.githubusercontent.com/32195694/31963924-a9ad243a-b902-11e7-9f19-279ac55e7838.png)

The property descripton does not exist.

Noe du kjenner til @olemp  ?

**Comment by pettersh** [comment]
Could it be that the Request Management Service in your farm is the root
cause of the list issue? The latest uls trace you posted in combination
with the event log screen shot indicates this.

24. okt. 2017 9:31 p.m. skrev "IngarMa" <notifications@github.com>:

Ser denne i en correlation:
[image: image]
<https://user-images.githubusercontent.com/32195694/31963924-a9ad243a-b902-11e7-9f19-279ac55e7838.png>

The property descripton does not exist.

Noe du kjenner til @olemp <https://github.com/olemp> ?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/Puzzlepart/prosjektportalen/issues/422#issuecomment-339105199>,
or mute the thread

**Comment by IngarMa** [comment]
@pettersh is request management service a requirement for the solution in a on-prem environment?

**Comment by tarjeieo** [comment]
Any update here @IngarMa ? Is it still failing?

**Comment by olemp** [comment]
Closing this @IngarMa. Add a new issue if needed. 👍
